### PR TITLE
A companion to: Updates to export-tool and experiment results 

### PR DESCRIPTION
**Pull request by samchrisinger**

### CODE REVIEW

File: exp-models/addon/models/session.js [hunk 1]
```diff
@@ -6,7 +6,9 @@ import Ember from 'ember';
 import DS from 'ember-data';
 import JamModel from '../mixins/jam-model';
 
-export default DS.Model.extend(JamModel, {
+const AnonJamModel = JamModel.without(['createdBy', 'modifiedBy']);
```
Comment (chrisseto) [inline exp-models/addon/models/session.js]: Neat

File: exp-models/addon/serializers/session.js [hunk 2]
```diff
@@ -4,5 +4,19 @@ import JamSerializer from '../mixins/jam-serializer';
 import JamDocumentSerializer from '../mixins/jam-document-serializer';
 
 export default DS.JSONAPISerializer.extend(JamSerializer, JamDocumentSerializer, {
-    modelName: 'session'
+    attrs: {
```
Comment (chrisseto) [inline exp-models/addon/serializers/session.js]: Isn't this in `JamSerializer` or `JamDocumentSerializer`?

File: exp-models/addon/serializers/session.js [hunk 3]
```diff
@@ -4,5 +4,19 @@ import JamSerializer from '../mixins/jam-serializer';
...
+        createdBy: {serialize: false},
+        modifiedOn: {serialize: false},
+        modifiedBy: {serialize: false}
+    },
+
+    modelName: 'session',
+    extractAttributes() {
+        var attributes = this._super(...arguments);
+        // Remove identifying information from the session model
+        delete attributes.createdBy;
```
Comment (chrisseto) [inline exp-models/addon/serializers/session.js]: Is this necessary?

File: exp-models/addon/models/session.js (hunk 1 above)
Comment (samchrisinger) [inline exp-models/addon/models/session.js]: Iknowrite

File: exp-models/addon/serializers/session.js (hunk 2 above)
Comment (samchrisinger) [inline exp-models/addon/serializers/session.js]: Yeah it can be deleted.

File: exp-models/addon/serializers/session.js (hunk 3 above)
Comment (samchrisinger) [inline exp-models/addon/serializers/session.js]: Yes

**Comment by abought** [comment]
After pulling newest develop and running bootstrap script on develop branch, I can no longer load the `/results` page due to an issue with the `sanitizeProfileId` field:

`Uncaught TypeError: Cannot read property 'split' of undefined`

I will investigate and follow up with a fix.

**Pull request merged**